qubesmanager/settings.py

```diff
@@ -926,27 +926,31 @@ def __init_devices_tab__(self):
         self.dev_list.add_all_button.setVisible(False)
         self.devices_layout.addWidget(self.dev_list)
 
-        devs = []
-        lspci = subprocess.check_output(['/usr/sbin/lspci']).decode()
-        for dev in lspci.splitlines():
-            devs.append((dev.rstrip(), dev.split(' ')[0]))
+        dom0_devs = [dev for dev in
+                     self.vm.app.domains['dom0'].devices['pci'].available()]
+
+        attached_devs = [dev for dev in
+                         self.vm.devices['pci'].persistent()]
```

Comment:
Those two could be simplified using `list()`, instead of list comprehension.

Reply:
why do you hate list comprehension????